contents/spotlight/startup-documenso.md

```diff
@@ -0,0 +1,79 @@
+
+“We track Star, Forks and Contributors on a weekly basis,” says Florent Merian from Documenso. “Stars are mainly useful to us as a social proof. They show that users trust us, and encourage others to do the same — but they also inspire us in the work we do!”
+
+Over the past year Documenso has put a lot of effort into growing its presence on GitHub, and learned several lessons about what does (and doesn’t) work...
+
+![Documenso GitHub Stars](../images/blog/documenso-stars.png)
+<Caption>Documenso has adopted various tactics to create a vibrant GitHub following</Caption>
+
+### 1. Start small, stay transparent
+
+Every journey starts with a single step, and for Documenso that was encouraging the team and their friends to star the repo. This alone helped them reach their first 50 GitHub stars. 
+
+“Those first 50 stars were mostly from friends and team members,” says Florent. “After that, we turned to the open source community.”
+
+The open source community was an obvious place for the team to focus initially, but what really provoked interest and engagement wasn’t the open source software — it was the open source _culture_.
+
+“We try to be as transparent as possible and share everything publicly, even revenue details,” says Florent. “This helped us engage with users on Twitter and, whenever we saw threads about open source software or the like we were able to get involved and link back to GitHub.”
+
+> **Putting it into practice:** Ask friends and family to star your repo, then widen the request to relevant communities you can reach out to. Closed source teams can find communities unique to their industry, and provoke conversations there. 
+
+### 2. Centralize the community 
+
+A hard-learned lesson for Documenso was that the team started off trying to build a following in Slack, which they came to realize wasn’t the best platform for their users. Eventually, they decided to migrate to Discord as a community hub.
+
+“Switching to Discord was a key moment for us,” says Florent. “We wanted to double down on our community building and make the whole project community driven — which meant we needed to move to a more developer-friendly environment.”
+
+As the community settled in, the team created topics for conversation and tried to encourage GitHub contributions and bug reports. This focus on GitHub was essential because, while Discord was a great place to engage users, GitHub was the _real_ heart of the project. 
+
+“We focus on Discord and Twitter now, but we want everything to link back to GitHub and compound the effects there. GitHub really is our main channel and our key focus. That’s where Documenso really lives. It doesn’t live on Documenso.com. It lives on GitHub.”
+
+> **Putting it into practice:** Do everything you can to direct users to GitHub, using CTAs and banners on other platforms. You can use issue templates and a comprehensive README to help users communicate with you and lower the barrier to entry. 
```

Comment:
You can use discussions and wiki pages, too (something we're experimenting with right now, cross-posting blog posts to GitHub).